src/v2/view-builder/internals/BaseView.js

```diff
@@ -14,9 +15,12 @@ export default View.extend({
 
   className: 'siw-main-view',
 
-  template: '<div class="siw-main-header"></div>' +
-      '<div class="siw-main-body"></div>' +
-      '<div class="siw-main-footer"></div>',
+  template: hbs`
+    <div class="siw-main-header"></div>
+    <div class="siw-callout"></div>
```

Comment:
why add `siw-callout` ?

Reply:
Good catch. I think this is probably a mistake from doing rebases/merges over the last couple months. I'll fix it.